docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+    application:
+      build:
+          dockerfile: ./Dockerfile
+          context: .
+      command: yarn watch
+      environment:
+          RAZOR_API_KEY: pleasechangeme
```

Comment:
oh sweet! didn't know docker-compose supported a .env file. I'll create a .env.dist and ignore a .env which will be used in the docker-compose :+1: